js/demo.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
```

Comment:
@kavanagh Apologies, you're absolutely right, I completely forgot you can create new DOM elements by just setting `innerHTML` and so was thinking I needed to parse the HTML template into a DOM before replacing on page. I've since replaced that with an implementation similar to what you suggest above.